metrics/metrics.go

```diff
@@ -39,11 +39,21 @@ var (
 		},
 		[]string{"type"},
 	)
+	// ScrappedErrorTotal is a Prometheus gauge to track the number of times that an scrapped aggregated error is produced.
+	ScrappedAggregatedErrorTotal = prometheus.NewGaugeVec(
```

Comment:
I think this needs to be a counter, not a gauge. Otherwise we won't be able to produce historical graphs

Reply:
I'm updating tha value every time that an scrappe is produced so a counter wouldn't work here

Reply:
Unless i move the logic here: https://github.com/soundcloud/periskop/blob/master/scraper/scraper.go#L25

Reply:
Also, it would look weird if in Prometheus there's more number of errors than in the Periskop UI.

Reply:
Ok, since the values can go down after a deploy, let's use a guage :+1: